erpnext/hr/doctype/leave_allocation/leave_allocation.js

```diff
@@ -5,20 +5,23 @@ cur_frm.add_fetch('employee','employee_name','employee_name');
 
 frappe.ui.form.on("Leave Allocation", {
 	onload: function(frm) {
+		// Ignore cancellation of doctype on cancel all.
+		frm.ignored_doctypes_on_cancel_all = ["Leave Ledger Entry"];
```

Comment:
I think it should be `ignore` instead of `ignored` since we are declaring the behavior.